docs/self-hosted/docker-guide.md

```diff
@@ -83,36 +83,27 @@ project:
 version: '3'
 services:
   database:
-    container_name: database
     image: postgis/postgis:13-master
     # Required when running on platform other than amd64, like Apple M1/M2:
     # platform: linux/amd64
     volumes:
       - ./data/database:/var/lib/postgresql/data
-    networks:
-      - directus
     environment:
       POSTGRES_USER: 'directus'
       POSTGRES_PASSWORD: 'directus'
       POSTGRES_DB: 'directus'
 
   cache:
-    container_name: cache
     image: redis:6
-    networks:
-      - directus
 
   directus:
-    container_name: directus
-    image: directus/directus:10.0.0
+    image: directus/directus:10.4.0
```

Comment:
Should the version be specific here? I wonder if it should be the major version or `latest`. 🤔 

```suggestion
    image: directus/directus:10
```

```suggestion
    image: directus/directus:latest
```

Reply:
We very specifically tell people to specify a version - we should do it here imo

Reply:
Because the `latest` tag is strange and works differently than most would expect i prefer to not start people off with this. https://vsupalov.com/docker-latest-tag/

Reply:
The main point here being the used REDIS variables are only available since `10.4` so this would break for anyone that has an older version cached locally as `latest` 🤓

Reply:
Yeah, right! A good reason for having a specific version 👍

Reply:
TIL, thanks all! 👍